### PR TITLE
Improve group creation UX

### PR DESCRIPTION
**Pull request by lukebarnard1**
- Open group settings when created for the first time
 - Add placeholder text to textarea indicating what sort of content should go there
 - Add a placeholder element when there is no long description set on the group

2017-11-06-183458_971x415_scrot
2017-11-06-183450_1015x347_scrot

Fixes https://github.com/vector-im/riot-web/issues/5522

### CODE REVIEW

File: src/i18n/strings/en_EN.json [hunk 1]
```diff
@@ -702,7 +702,9 @@
     "You are a member of this community": "You are a member of this community",
     "Community Member Settings": "Community Member Settings",
     "Publish this community on your profile": "Publish this community on your profile",
+    "Your community hasn't got a Long Description, a HTML page to show to community members.<br />Click here to open settings and give it one!": "Your community hasn't got a Long Description, a HTML page to show to community members.<br />Click here to open settings and give it one!",
     "Long Description (HTML)": "Long Description (HTML)",
+    "<h1>HTML for your community's page</h1>\n<p>\n    Use the Long Description to introduce new members to the community, or distribute some important <a href=\"https://about.riot.im\">links</a>.\n</p> \n<p> \n    You can even use \"img\" tags, but only for images reachable from MXC URLs.\n</p>\n": "<h1>HTML for your community's page</h1>\n<p>\n    Use the Long Description to introduce new members to the community, or distribute some important <a href=\"https://about.riot.im\">links</a>.\n</p> \n<p> \n    You can even use \"img\" tags, but only for images reachable from MXC URLs.\n</p>\n",
```
Comment (turt2live) [inline src/i18n/strings/en_EN.json]: Are the `\n`'s intended?

Comment (lukebarnard1) [reply src/i18n/strings/en_EN.json]: yeup, the textarea obeys them

File: src/components/structures/GroupView.js [hunk 2]
```diff
@@ -862,6 +879,16 @@ export default React.createClass({
                 <h3> { _t("Long Description (HTML)") } </h3>
                 <textarea
                     value={this.state.profileForm.long_description}
+                    placeholder={_t(
```
Comment (ara4n) [inline src/components/structures/GroupView.js]: I wonder if this monster would be easier to read as a constant which can be defined at the top level somewhere?  e.g.:

```
const LONG_DESC_PLACEHOLDER = _t("<h1>HTML for your community's page</h1>
<p>
    Use the long description to introduce new members to the community, or distribute
    some important <a href="foo">links</a>
</p>
<p>
    You can even use 'img' tags
</p>
";
'''

...feels more legible to me?

Comment (lukebarnard1) [reply src/components/structures/GroupView.js]: wfm

File: src/i18n/strings/en_EN.json (hunk 1 above)
Comment (ara4n) [reply src/i18n/strings/en_EN.json]: yup, based on the _t() contents

**Comment by ara4n** [comment]
lgtm mod comments

File: src/i18n/strings/en_EN.json (hunk 1 above)
Comment (ara4n) [reply src/i18n/strings/en_EN.json]: NEEDS MOAR \n

**Pull request merged**